compiler-rt/lib/scudo/standalone/condition_variable_base.h

```diff
@@ -0,0 +1,57 @@
+//===-- condition_variable_base.h ------------------------------------*- C++
+//-*-===//
```

Comment:
Looks like this got split across lines on accident.

Reply:
Nice Catch!